src/Console/Command/FixCommand.php

```diff
@@ -306,7 +308,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->stopwatch->stop('fixFiles');
         $this->eventDispatcher->removeListener(FixerFileProcessedEvent::NAME, [$progressOutput, 'onFixerFileProcessed']);
 
-        $progressOutput->printLegend();
+        if (\count($changed) > 0) {
+            $progressOutput->printLegend();
+        }
```

Comment:
This is done because for `meaningful` output if no file is modified, then there's no progress output and legend printed just after "loaded config (...)" looks weird. But maybe it should be based on `instanceof` or other thing.

Reply:
Looks good to me.